submariner-operator/issue-149-node-placement.md

```diff
@@ -63,13 +63,6 @@ type SubmarinerDiscoverySpec struct {
 }
 ```
 
-Add option to `subctl join` to specify node selectors and tolerations.
```

Comment:
This exactly answers my query here https://github.com/submariner-io/enhancements/pull/156#issuecomment-1446143690. Don't know why this was removed. But I guess others have understood and don't want to block this.